src/components/Cluster/ClusterDetail/NodePool.tsx

```diff
@@ -210,14 +288,21 @@ class NodePool extends Component<INodePoolsProps, INodePoolsState> {
             <NodesWrapper data-testid='scaling-min'>{scaling.min}</NodesWrapper>
             <NodesWrapper data-testid='scaling-max'>{scaling.max}</NodesWrapper>
             <NodesWrapper>{desired}</NodesWrapper>
-            <NodesWrapper
-              style={{
-                background:
-                  current < desired ? theme.colors.goldBackground : undefined,
-              }}
-            >
-              {current}
-            </NodesWrapper>
+            <NodesWrapper highlight={current < desired}>{current}</NodesWrapper>
+            {typeof spot_instances !== undefined || spot_instances !== null ? (
```

Comment:
This can be just this, right?
```suggestion
            {spot_instances ? (
```

Reply:
does not work for `spot_instances === 0`

Reply:
Oh ok, did not check the data type, thought it was an object 👍

Reply:
tuns out I am a little dummy boy :) this should've been `typeof spot_instances !== 'undefined' && spot_instances !== null`. But I'll change it to `typeof spot_instances === 'number'`